content/v2/openapi.yml

```
@@ -2697,6 +2697,107 @@ paths:
         '204':
           description: Successfully deleted a webhook.
       summary: Delete a webhook
+  '/{account}/dns_analytics':
+    get:
+      description:
+        Queries and returns DNS Analytics data available for the provided query parameters.
+        This API is currently in Public Beta. During the Public Beta period changes may occur at any time.
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - name: start_date
+          description:
+            Only include results after the given date. Format is ISO8601 (YYYY-MM-DD).
+            It can be at most 31 days away from the end_date.
+            If omitted it defaults to 31 days ago.
+          style: form
+          in: query
+          required: false
+          schema:
+            $ref: '#/components/schemas/Date'
+        - name: end_date
+          description:
+            Only include results before the given date. Format is ISO8601 (YYYY-MM-DD).
+            It can be at most 31 days away from the start_date.
+            If omitted it defaults to 1 day ago.
+          style: form
+          in: query
+          required: false
+          schema:
+            $ref: '#/components/schemas/Date'
+        - name: groupings
+          description:
+            How to group the results about the account's DNS analytics.
+            No groupings will be applied by default unless specified.
+            Multiple groupings can be provided, separated by a comma.
+            Applying groupings to the query will add columns to the response payload.
+          style: form
+          in: query
+          required: false
+          schema:
+            type: string
+            enum:
+              - 'date'
+              - 'volume'
+              - 'zone_name'
+        - $ref: '#/components/parameters/SortDNSAnalytics'
+        - $ref: '#/components/parameters/PaginationPage'
+        - $ref: '#/components/parameters/PaginationPerPage'
+      responses:
+        '200':
+          description: Successfully retrieved DNS analytics data.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                    properties:
+                      rows:
+                        type: array
+                        items:
+                          anyOf:
+                            - type: integer
+                            - type: string
+                            - $ref: '#/components/schemas/Date'
+                          example: [example.com,102985]
```

```suggestion
                          example: ["example.com", "2024-08-01", 102985]
```
A couple of things:
- Add a date for a complete example
- Use double quote wrapper for strings for JSON compliance